### ISSUE TITLE
Implement irssi parsing/formatting

### ISSUE THREAD

**Issue by tilpner** [tilpner/ilc#6]
WIP in formats/irssi

**Comment by ailin-nemui** [comment]
keep in mind that irssi can be completely themed so possibly at best you can support the default.theme log format

**Comment by tilpner** [comment]
Yes, of course, I don't plan to support customization of logs at all, although it would be possible.
The current weechat parser relies on the standard date format as well, but... really, why would you even change that? :/
